ClinGen/clincoded
# Connects to #1536. Adding new users.

**Technical notes:**
Prior to verifying `Orphanet:154` not being migrated to MONDO ID, production data needs to be loaded into the test instance and then the batchupgrade script has been run.

**Steps to test:**
1. Go to ^/users/.
2. Enter either **Tawari** or **Hasadsri** in the **_Filter table by_** search box at the upper right hand corner.
3. Expect to see a filtered entry being returned for the user.
4. Go to ^/diseases/.
5. Enter **Orphanet_154** in the **_Filter table by_** search box at the upper right hand corner.
6. Expect to see a filtered entry being returned for the disease Id.
